<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 89, 156);</string>
  </property>
  <widget class="QPushButton" name="ButtonMedici">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/IconaMedico.png</normaloff>../../../../MATERIALE/IconaMedico.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>140</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="shadow" stdset="0">
    <string/>
   </property>
   <property name="set" stdset="0">
    <UInt>0</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="FrameBianco">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>0</y>
     <width>540</width>
     <height>450</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New'; background-color:#2b2b2b;&quot;&gt;&lt;br/&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../MATERIALE/InterfacciaAmministratore/BackGroundAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInfermieri">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(230, 235, 239);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/IconaInfermiere.png</normaloff>../../../../MATERIALE/IconaInfermiere.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="shadow" stdset="0">
    <string/>
   </property>
   <property name="set" stdset="0">
    <UInt>0</UInt>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLogout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>405</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(230, 235, 239);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/Logout sinistra.png</normaloff>../../../../MATERIALE/Logout sinistra.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="shadow" stdset="0">
    <string/>
   </property>
   <property name="set" stdset="0">
    <UInt>0</UInt>
   </property>
  </widget>
  <zorder>FrameBianco</zorder>
  <zorder>ButtonMedici</zorder>
  <zorder>ButtonInfermieri</zorder>
  <zorder>ButtonLogout</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
